site/src/components/popover.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Popover
+description: The Popover component in the Atlas Design System
+template: standard
+---
+
+# Popover
+
+The popover component allows both simple and more complex content to be placed in the popup that appears after clicking on a button. Popover is based on top of the [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#customizing_the_disclosure_widget) HTML component.
+
+```html
+<details class="popover breadcrumbs-overflow" data-popover-details>
```

Comment:
What's the usage of the `data-popover-` attributes for?

Reply:
First of all, it was they were in the suggested template from Will. Secondly, I thought to use it for the targeting elements to add event listeners (which I'm going to add in docs). But I can do that by pulling the class name. Will do some clean up

Reply:
If I recall our conversation from a few weeks ago (high likelihood I'm misremembering!), I think we discussed trying to do a web component first.

Reply:
I'm any case the main thing we need in this PR is the css. We should try and reduce extra information in examples as much as possible.

Reply:
I don't remember the details either tbh, so I'll clean it up, and if needed - I'll add it back later

Reply:
Cleaned up!